test/gtest/uct/test_md.cc

```diff
@@ -813,6 +813,35 @@ UCS_TEST_SKIP_COND_P(test_md, dereg_bad_arg,
     free(ptr);
 }
 
+UCS_TEST_SKIP_COND_P(test_md, exported_mkey,
+                     !check_caps(UCT_MD_FLAG_EXPORTED_MKEY))
+{
+    static const size_t size       = 1 * UCS_MBYTE;
+    static const unsigned md_flags = UCT_MD_MEM_ACCESS_REMOTE_PUT |
+                                     UCT_MD_MEM_ACCESS_REMOTE_GET;
+    uct_mem_h export_memh;
+    void *ptr;
+    ucs_status_t status;
+
+    ptr    = malloc(size);
+    status = reg_mem(md_flags | UCT_MD_MEM_FLAG_EXPORT, ptr, size,
+                     &export_memh);
+    ASSERT_UCS_OK(status);
+
+    std::vector<uint8_t> mkey_buffer(/* Replace by md_attr().exported_mkey_packed_size*/
+                                     md_attr().rkey_packed_size);
+    uct_md_mkey_pack_params_t pack_params = { 0 };
+    status                                =
+            uct_md_mkey_pack_v2(md(), export_memh, &pack_params,
+                                mkey_buffer.data());
```

Comment:
is it clang suggestion? imo, looks weird

Reply:
still looks weird .. add space line before